components/Table/Table.tsx

```diff
@@ -48,7 +47,6 @@ export const Table: FC<TableProps> = ({
 	className,
 	showPagination = false,
 	pageSize = null,
-	hiddenColumns = undefined,
```

Comment:
@JChiaramonte7 I checked the shorting page. There is a shorting history table using this component and undefined is not an acceptable data type. We have to pass an empty array here, like this ```hiddenColumns = []```. How do you want to fix this issue?